web/js/libs/browser_utils.js

```diff
@@ -4,27 +4,7 @@
 
   var getCurrentTime = function () {
     var now = new Date();
-    var time = [];
-    var suffix = ' AM';
-
-    var hours = now.getHours();
-    if (hours >= 12) {
-      suffix = ' PM';
-    }
-    if (hours > 12) {
-      hours -= 12;
-    }
-
-    time.push(hours);
-    time.push(':');
-
-    var minutes = now.getMinutes();
-    minutes < 10 && time.push('0');
-
-    time.push(minutes);
-    time.push(suffix);
-
-    return time.join('');
+    return now.toLocaleTimeString('en-US', {hour: '2-digit', minute:'2-digit'});
```

Comment:
This is a minor comment but how about:
`return new Date().toLocaleTimeString('en-US', {hour: '2-digit', minute:'2-digit'});`

one liner 😎

Reply:
@robjperez done!
For some reason I thought you could create a date and format it in one line, but you can so..